### ISSUE TITLE
Live streaming delay

### ISSUE THREAD

**Issue by rockstardev8964** [TakuSemba/RtmpPublisher#36]
I am using your code in my local. But the streaming is delaying about 20 ~ 30s.
How can I reduce this delaying time?

**Comment by chrishobcroft** [comment]
Can you be more clear what you mean?

Are you saying that there is a delay between the time you send video, and the time it plays back video?

**Comment by rockstardev8964** [comment]
Yes, the time between sending and playing back is more than 30s.
How can I decrease this time?
________________________________
